frontend/src/components/Secrets/GSecretDialogDDNS.vue

```diff
@@ -0,0 +1,245 @@
+<!--
+SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+ -->
+
+<template>
+  <g-secret-dialog
+    v-model="visible"
+    :data="secretData"
+    :secret-validations="v$"
```

Comment:
Why do we pass the complete vuelidate object to another component. This looks very strange.

Reply:
This because of the parent has the fields that the child component needs to validate. The components are stacked the wrong way. Just discussed this with Peter. We should fix this. But it makes sense to merge #1553 first, then get rid of all obsolete dialogs and then cleanup this mess.

Reply:
Yes.